testng-core/src/main/java/org/testng/internal/ClonedMethod.java

```diff
@@ -253,9 +252,7 @@ public boolean isTest() {
   }
 
   @Override
-  public void setDate(long date) {
-    m_date = date;
-  }
+  public void setDate(long ignored) {}
```

Comment:
Why is it ignored now?

Reply:
Because it's not used. The setter is setting a data member which is not being read. The `getDate()` actually returns back the respective value from embedded `m_method`

Reply:
Why not throwing an exception as it is not supposed to be used?

Reply:
This is not external facing. It is being used internally. So not sure what is the point of throwing an exception here. I believe this class was just intended to act as a wrapper for an existing `ITestNGMethod` but since it has to adhere to the interface construct, it has to implement this method. Throwing the exception is perhaps not going to solve anything here.

Reply:
Why not calling set method from the internal object?

My point is the behavior is unpredictable if a value is passed and ignored.

Reply:
Ok Got it. So you basically would like that call to be passed on as well by the wrapper into the wrapped object. Fixed and pushed the change